### PR TITLE
Added my Tile (Yellow) :D

### PR DESCRIPTION
**Pull request by Beta-King**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]

### My steps to solve it,
[Mention the files changed. Add what changes you have done.] 

### Screenshots,
[If any]

### Links

http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

**Comment by Beta-King** [comment]
@Ploypiti how do i add it - could you please guide me :D

**Comment by Beta-King** [comment]
Is it ohk now ??

**Comment by saarthakchats** [comment]
Can't find your tile?

**Comment by samyakbvs** [comment]
Pls squash your commits

**Comment by saarthakchats** [comment]
Please give a screenshot of the tile.

**Comment by Ploypiti** [comment]
I can't see your tile, it should be here;

ss

**Comment by Beta-King** [comment]
@Ploypiti how do i make the necessary changes then ?? can you help me :D

**Comment by Ploypiti** [comment]
@Beta-King Just replied to your dm.

File: tiles/rooms/wall/ColorWallTop.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
...AAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAwCcC/wd4S3JjNvd7DQAAAABJRU5ErkJggg=="></image>
```
Comment (niccokunzmann) [inline tiles/rooms/wall/ColorWallTop.svg]: As I am looking at your rectangle, I would like to know what is behind it what is in this image?

**Pull request closed without merging**